fundingmanager.go

```diff
@@ -1388,6 +1418,15 @@ func (f *fundingManager) handleFundingAccept(fmsg *fundingAcceptMsg) {
 		return
 	}
 
+	// Set the remote shutdown script on the reservation. If the upfront shutdown
+	// script is empty, this call will have no effect.
+	if err := resCtx.reservation.ProcessUpfrontShutdown(
```

Comment:
name `SetUpfrontShutdown` instead? TO indicate we are not processing a new remote message.